packages/camera/camera/test/camera_test.dart

```diff
@@ -1180,6 +1181,31 @@ void main() {
       expect(cameraController.value.isPreviewPaused, equals(true));
     });
 
+    test(
+        'pausePreview() sets previewPauseOrientation according to locked orientation',
+        () async {
+      final CameraController cameraController = CameraController(
+          const CameraDescription(
+              name: 'cam',
+              lensDirection: CameraLensDirection.back,
+              sensorOrientation: 90),
+          ResolutionPreset.max);
+      await cameraController.initialize();
+      cameraController.value = cameraController.value.copyWith(
+          isPreviewPaused: false,
+          deviceOrientation: DeviceOrientation.portraitUp,
+          lockedCaptureOrientation:
+              const Optional<DeviceOrientation>.fromNullable(
```

Comment:
nit: Same for this one. Should this be using `.of`.

```suggestion
              const Optional<DeviceOrientation>.of(
```